packages/dashboard/src/components/SetupBanner/SetupBanner.tsx

```diff
@@ -15,36 +15,27 @@ export class SetupBanner extends React.Component<ISetupBannerProps, {}> {
         <div className={classNames.textPartition}>
           <div className={classNames.headerSection}>{headerText}</div>
           <div className={classNames.bodySection}>{onRenderBody()}</div>
-          {this._renderActions()}
+          <div className={classNames.actionSection}>
+            {this.props.actions.map((action: ISetupBannerAction, actionIndex: number) => {
+              return this._renderAction(action, actionIndex);
+            })}

```

Comment:
_I think_ you can collapse this logic even further as:

```ts
this.props.actions.map(this._renderAction)
```

I don't think you'll run into a scoping issue. Just a consideration.

Reply:
You would be correct, thanks for the suggestion!